build/npm-build-extensions-steps.yml

```diff
@@ -0,0 +1,26 @@
+# Steps to build and test npm project
+
+steps:
+    - task: NodeTool@0
+      inputs:
+          versionSpec: '10.x'
+      displayName: 'Install Node.js'
+
+    - task: Npm@1
+      displayName: 'npm ci'
```

Comment:
Is this running these commands against the entire repo?

Reply:
Not anymore. It only runs against the extensions (line 13)

Reply:
Ah right missed that.

Reply:
I just added it today